openxla-nvgpu/compiler/src/openxla/compiler/nvgpu/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+iree_lit_test_suite(
+        NAME
```

Comment:
Indentation is off, I think it's two spaces everywhere

Reply:
Done.